pallet-fuso-verifier/Cargo.toml

```diff
@@ -24,15 +24,24 @@ sp-core = { default-features = false, git = "https://github.com/paritytech/subst
 sp-io = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13"  }
 sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13"  }
 fuso-support = { path = "../fuso-support", default-features = false }
+pallet-fuso-token = { path = "../pallet-fuso-token", default-features = false }
+pallet-balances = { git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "polkadot-v0.9.13"}
 smt = { git = "https://github.com/uinb/sparse-merkle-tree", tag = "v0.1.7", package = "sparse-merkle-tree", default-features = false, features = ["blake2b"] }
+hex = { version = "0.4", default-features = false }
 
```

Comment:
since you just add benchmark, you shouldn't add dependencies directly. set optional=true  with runtime-benchmark feature

Reply:
done